runtime/devnet/src/lib.rs

```diff
@@ -955,12 +942,16 @@ impl_runtime_apis! {
 	}
 
 	impl sp_genesis_builder::GenesisBuilder<Block> for Runtime {
-		fn create_default_config() -> Vec<u8> {
-			create_default_config::<RuntimeGenesisConfig>()
+		fn build_state(config: Vec<u8>) -> sp_genesis_builder::Result {
+			build_state::<RuntimeGenesisConfig>(config)
+		}
+
+		fn get_preset(id: &Option<sp_genesis_builder::PresetId>) -> Option<Vec<u8>> {
+			get_preset::<RuntimeGenesisConfig>(id, |_| None)
 		}
 
-		fn build_config(config: Vec<u8>) -> sp_genesis_builder::Result {
-			build_config::<RuntimeGenesisConfig>(config)
+		fn preset_names() -> Vec<sp_genesis_builder::PresetId> {
```

Comment:
We might be missing the chance to move genesis config from the node to the runtime. For instance as in [here](https://github.com/polkadot-fellows/runtimes/blob/ca6992420f08731b2a683bfa893354b72a15848f/relay/polkadot/src/genesis_config_presets.rs#L228).

Reply:
Just notice that part, I will fix it. Thanks for reporting it

Reply:
Created an issue: https://github.com/r0gue-io/pop-node/issues/196